packages/elements/src/appstate-bar/__test__/appstate-bar.test.js

```diff
@@ -60,7 +60,7 @@ describe('appstate-bar/AppstateBar', () => {
     );
   });
 
-  it('Should changed heading background to default color when invalid state is set', () => {
+  it('Should changed heading background to default color when invalid state is set', function() {
```

Comment:
Should we add `elementUpdated(el)` after line 70 ?

Reply:
good catch! Fixed.